lib/controllers/Motor.cpp

```diff
@@ -23,15 +23,14 @@ void Motor::setPower(float dutyCycle) {
   m_pwm = fmin(fabs(dutyCycle), m_limit);
 }
 
+// TODO: this function results in a warning "control reaches end of non-void function", not sure how to fix it
```

Comment:
Not exactly sure either, though this function is never used and is not very explicit so you can remove it.

Reply:
Usually this error occurs when you don't return a value for a non-void func

Reply:
Sorry I think this comment was left over from an earlier version ... I don't think we have this warning anymore.